src/main/java/org/openelisglobal/login/valueholder/UserSessionData.java

```diff
@@ -31,6 +33,7 @@ public class UserSessionData implements Serializable {
     private int systemUserId;
     private String loginName;
     private boolean isAdmin;
+    private  LoginUserService loginService  = SpringContext.getBean(LoginUserService.class);
```

Comment:
@gitcliff , 
this is also Anti pattern. you dont want to mix the service classes into the dto objects
can you pass the expiry date string ,from where the other attributes are passed from ??